app/src/main/java/com/github/nf1213/choreapp/ChoreViewHolder.java

```diff
@@ -22,12 +24,13 @@
     public ChoreViewHolder(View itemView, ChoreAdapter.ClickListener clickListener, ChoreAdapter.DeleteListener deleteListener, ChoreAdapter.CheckListener checkListener) {
         super(itemView);
         mTextView = (TextView) itemView.findViewById(R.id.name);
+        mEdit = (Button) itemView.findViewById(R.id.edit);
         mCheckBox = (CheckBox) itemView.findViewById(R.id.checkbox);
         mDeleteListener = deleteListener;
         mClickListener = clickListener;
         mCheckListener = checkListener;
         mTextView.setOnLongClickListener(ChoreViewHolder.this);
-        mTextView.setOnClickListener(this);
```

Comment:
I addressed all of your other comments, perhaps this can be the next task I work on for this project?